elements/rh-tag/demo/filled-color-with-icon.html

```diff
@@ -6,7 +6,7 @@
 <rh-tag icon="information-fill" color="teal">Teal</rh-tag>
 <rh-tag icon="information-fill" color="blue">Blue</rh-tag>
 <rh-tag icon="information-fill" color="purple">Purple</rh-tag>
-<rh-tag icon="information-fill" color="grey">Gray</rh-tag>
+<rh-tag icon="information-fill" color="gray">Gray</rh-tag>
```

Comment:
Should this change have been considered a breaking change?   Should we have deprecated the `gray` first in favor of `grey`?